src/Components/DataEntry/FileInput/FileSummary.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<div class="c-application c-file--summary">
+		<div class="c-file--summary-left-content-wrapper">
```

Comment:
```suggestion
		<div class="c-file--summary-info-wrapper">
```
클래스명을 이렇게 짓는건 어떨까요? summary의 요소는 정보영역, 버튼영역 2개니까 left라고 하지않아도 구분될 것 같아요.

Reply:
@yooseongsil 
더 좋네요! 작성당시에 아이디어가 안 떠올랐네요